tortoise/models.py

```diff
@@ -614,7 +614,9 @@ def __search_for_field_attributes(base: Type, attrs: dict) -> None:
         meta.pk_attr = pk_attr
         meta.pk = fields_map.get(pk_attr)  # type: ignore
         if meta.pk:
-            meta.db_pk_column = meta.pk.source_field or meta.pk_attr
+            meta.db_pk_column = meta.pk.source_field or (
+                meta.pk_attr + "_id" * isinstance(meta.pk, OneToOneFieldInstance)
```

Comment:
May be we should just do it like that?

```
if meta.pk.source:
    meta.db_pk_column = meta.pk.source_field
elif isinstance(meta.pk, OneToOneFieldInstance):
    meta.db_pk_column = f"{meta.pk_attr}_id"
else:
    meta.db_pk_column = meta.pk_attr
```

Yeah, it's a bit more verbose, but I think trying to squeeze 3 outcomes in one expression doesn't really look all that good and readable
```